app/controllers/picks_controller.rb

```diff
@@ -0,0 +1,71 @@
+class PicksController < ApplicationController
+  before_action :authenticate, :commissioner_authenticate
+  def new
+    binding.pry
+  end
+  def create
```

Comment:
Perceived complexity for create is too high. [9/7]<br>Use empty lines between defs.